src/BloomExe/web/ReadersHandler.cs

```diff
@@ -255,6 +258,13 @@ private static void MakeLetterAndWordList(string jsonSettings, string allWords)
 			PathUtilities.OpenFileInApplication(fileName);
 		}
 
+		private static string GetLocalizedString(string key, string defaultText)
+		{
+			return LocalizationManager.UILanguageId == "en"
+				? LocalizationManager.GetDynamicString("Bloom", key, defaultText)
```

Comment:
Sorry, I seem to be getting baffled by one thing after another...why do we need this special case for English?